app/controllers/hyrax/file_sets_controller.rb

```diff
@@ -254,12 +225,6 @@ def empty_file?(file)
         (file.respond_to?(:tempfile) && file.tempfile.size == 0) || (file.respond_to?(:size) && file.size == 0)
       end
 
-      # this is provided so that implementing application can override this behavior and map params to different attributes
```

Comment:
They may seem unrelated, but I believe they're being used by folks to make overrides such as:

* https://github.com/duke-libraries/course-assets/blob/2c8a64687649eb2ebc8cf6777edf270ada1e3ed2/lib/course_assets/files_controller_behavior.rb#L5-L7
* https://github.com/ualbertalib/HydraNorth/blob/d4a1c87cf34e376f69138c411cc21819f1edb2f2/app/controllers/concerns/hydranorth/files_controller_behavior.rb#L61-L65
* https://github.com/ihrnexuslab/research-repo/blob/572865c2125c18891762abaa0efdeb2d9895da7d/app/controllers/generic_files_controller.rb#L121-L132
* https://github.com/sephirothkod/Sufia-Head/blob/7994b6eb6eeb29ac40b3d243af8f65bc07d43947/app/controllers/concerns/sufia/files_controller_behavior.rb#L286-L297

Given that folks are still using this seam, and it's not necessary for this PR (methinks?), would you consider reverting this change?